scripts/code-examples.js

```diff
@@ -0,0 +1,133 @@
+const fs = require('fs-extra');
```

Comment:
Currently on the monorepo branch, all the scripts have been rewritten to `mjs`, I think this one should too, to keep things consistent.

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/c51d6a8f1754508d50e53c11a3d50130724d9cd1